Milvus.Client.Tests/SearchQueryTests.cs

```diff
@@ -531,6 +531,151 @@ await collection.WaitForCollectionLoadAsync(
+        var int16Field = (FieldData<int>)results.FieldsData.Single(f => f.FieldName == "int16");
+        Assert.Equal(new[] { 1, 2 }, int16Field.Data);
+        var int32Field = (FieldData<int>)results.FieldsData.Single(f => f.FieldName == "int32");
+        Assert.Equal(new[] { 1, 2 }, int32Field.Data);
+        var int64Field = (FieldData<long>)results.FieldsData.Single(f => f.FieldName == "int64");
+        Assert.Equal(new[] { 1L, 2L }, int64Field.Data);
+        var floatField = (FieldData<float>)results.FieldsData.Single(f => f.FieldName == "float");
+        Assert.Equal(new[] { 1.1f, 2.2f }, floatField.Data);
+        var doubleField = (FieldData<double>)results.FieldsData.Single(f => f.FieldName == "double");
+        Assert.Equal(new[] { 1.1, 2.2 }, doubleField.Data);
+        var varcharField = (FieldData<string>)results.FieldsData.Single(f => f.FieldName == "varchar");
+        Assert.Equal(new[] { "one", "two" }, varcharField.Data);
+        var boolArrayField = (ArrayFieldData<bool>)results.FieldsData.Single(f => f.FieldName == "bool_array");
+        Assert.Equal(new[] { new[] { true }, new bool[] { } }, boolArrayField.Data);
+        var int8ArrayField = (ArrayFieldData<int>)results.FieldsData.Single(f => f.FieldName == "int8_array");
+        Assert.Equal(new[] { new[] { 1 }, new int[] { } }, int8ArrayField.Data);
+        var int16ArrayField = (ArrayFieldData<int>)results.FieldsData.Single(f => f.FieldName == "int16_array");
+        Assert.Equal(new[] { new[] { 1 }, new int[] { } }, int16ArrayField.Data);
+        var int32ArrayField = (ArrayFieldData<int>)results.FieldsData.Single(f => f.FieldName == "int32_array");
+        Assert.Equal(new[] { new[] { 1 }, new int[] { } }, int32ArrayField.Data);
+        var int64ArrayField = (ArrayFieldData<long>)results.FieldsData.Single(f => f.FieldName == "int64_array");
+        Assert.Equal(new[] { new[] { 1L }, new long[] { } }, int64ArrayField.Data);
+        var floatArrayField = (ArrayFieldData<float>)results.FieldsData.Single(f => f.FieldName == "float_array");
+        Assert.Equal(new[] { new[] { 1.1f }, new float[] { } }, floatArrayField.Data);
+        var doubleArrayField = (ArrayFieldData<double>)results.FieldsData.Single(f => f.FieldName == "double_array");
+        Assert.Equal(new[] { new[] { 1.1 }, new double[] { } }, doubleArrayField.Data);
+        var varcharArrayField = (ArrayFieldData<string>)results.FieldsData.Single(f => f.FieldName == "varchar_array");
+        Assert.Equal(new[] { new[] { "one" }, new string[] { } }, varcharArrayField.Data);
+        var jsonField = (FieldData<string>)results.FieldsData.Single(f => f.FieldName == "json");
+        Assert.Equal(new[] { "{}", "{\"a\":1}" }, jsonField.Data);
+        var binaryVectorField = (BinaryVectorFieldData)results.FieldsData.Single(f => f.FieldName == "binary_vector");
+        Assert.Equal((ReadOnlyMemory<byte>)new byte[] { 0x00, 0xFF }, binaryVectorField.Data[0]);
```

Comment:
```csharp
FieldData.CreateBinaryVectors("binary_vector", new[]
{
    (ReadOnlyMemory<byte>)new byte[] { 0x00 },
    (ReadOnlyMemory<byte>)new byte[] { 0xFF },
}),
```
2 elements were added but now it's only one element with the values merged. That's weird right?

Reply:
I confirm that discarding the changes in `CreateFromBytes` makes the tests `Create_binary_vector_field()`, `Search_binary_vector` fail.